src/Confluent.Kafka/Impl/NativeMethods/NativeMethods_Alpine.cs

```diff
@@ -40,7 +40,11 @@ namespace Confluent.Kafka.Impl.NativeMethods
     /// </remarks>
     internal class NativeMethods_Alpine
     {
+#if NET45 || NET46 || NET47
+         public const string DllName = "alpine-librdkafka.so";
```

Comment:
prior to this PR, we would never try to load librdkafka in the NET45/46/47 case on linux (which is what mono allows you to do), so this change does not modify any scenario that was currently operational.